pym/bob/scm/scm.py

```diff
@@ -60,6 +65,15 @@ def __str__(self):
                 + (("set: " + str(self.__set)+ "\n") if self.__set else "")
                 + (("replace: " + str(self.__replaceRaw)) if self.__replaceRaw else "")).rstrip()
 
+
+    def __substitute(self, env):
+        for attr in ['__match', '__del', '__set', '__replaceRaw']:
+            i = getattr(self, "_ScmOverride"+attr)
```

Comment:
:scream: Uhh, that works but is almost unreadable for people who are not Python pro's... :wink:

Reply:
:disappointed: it took me some time to google that :wink: